### PR TITLE
Resolution 2021-01

### PR DESCRIPTION
**Pull request by DuaneOBrien**

### CODE REVIEW

File: resolutions/numbers.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 [2020-01](approved/2020-01.md) - Formation of Charter Maintenance Working Group
 
 [2020-02](proposed/2020-02.md) - Formation of Outreach Working Group
+
+[2021-01](approved/2021-01.md) - Clarify Steering Committee's responsibility to oversee business and marketing matters
```
Comment (gtback) [inline resolutions/numbers.md:7]: Right now this is in the `proposed` directory. I'm it will get moved if the resolution is approved?

Comment (DuaneOBrien) [reply resolutions/numbers.md:7]: Correct.

**Comment by geekygirldawn** [comment]
This looks good to me, and thank you for clarifying this, since it's been a bit of a pain point for the SC in the past!

**Comment by kpfleming** [comment]
Approved by me.

**Comment by ashleywolf** [comment]
+1 I think this is a needed clarifier

**Comment by geekygirldawn** [comment]
@anajsana and others - it looks like this has several approvals already - what do we need to do to get this merged?

**Comment by anajsana** [comment]
I've added a new comment to the document. I think we should be even more specific about Steering Committee's planning level and the process this would follow. For instance:

1) The Steering Committee should act on a strategic level: SC Members should define and approve a set of strategic goals for each year during the SC monthly meeting, that will be documented at: https://github.com/todogroup/governance/blob/master/goals.md

2) Program Manager should act on a tactical level: PM should evaluate those goals and define and execute initiatives aligned with the strategic goals for that year. Only if an initiative does not fit in any of the defined strategic goals, the initiative should be reviewed and approved by the SC.

3) SC should be informed and provide advice on existing and new initiatives that are aligned with the strategic goals during the SC meetings.

This approach would avoid clogging up the design and execution of new initiatives and keep the project moving forward under the SC strategic vision.

File: resolutions/proposed/2021-01.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Subject: Clarify Steering Committee's responsibility to oversee business and marketing matters
+
+On 2021-09-14, the Steering Committee of the TODO Group reviewed and {approved/declined} the following resolution:
+
+Amend section 3.a.ii to clarify the responsibility of the Steering Committee to oversee business and marketing matters.
+
+3.a.ii Previous Language: overseeing all business and marketing matters;
+
+3.a.ii Proposed Language: overseeing all business and marketing matters, including approving new programs and initiatives that make use of the TODO Group name, logo, or branding, and approving use of the TODO Group name, logo, or branding for sponsorship or endorsement purposes;
```
[the remark this answers is not in the document]
Comment (DuaneOBrien) [reply resolutions/proposed/2021-01.md:13]: I don't disagree with the clarification, but I think we should handle it with a separate charter amendment.

**Comment by DuaneOBrien** [comment]
We have consensus on +1s, I'm going to merge forward, capture the approvals, and open an issue that reflects the need to better document the expectations and responsibilities of the LF PM assigned to support TODO

**Pull request merged**